website/versioned_docs/version-0.10.0/ergo-intro-examples.md

```diff
@@ -23,7 +23,7 @@ This declares that this module belongs to the `org.accordproject.helloworld` nam
 
 It also declares that the contract `HelloWorld` is parameterized over a given `TemplateModel`.
 
-The `TemplateModel` as well as the `Request` and `Response` are types which are specified using [Composer Concerto modeling language](https://github.com/hyperledger/composer-concerto).
+The `TemplateModel` as well as the `Request` and `Response` are types which are specified using [Composer Concerto modeling language](https://github.com/accordproject/concerto).
```

Comment:
This is a static versioned file and shouldn't be edited, please remove this change.

When we make a new release of the documentation, Docusarus creates these `versions_docs` automatically as a snapshot of the documentation for historical reference.